lib/wings/active_fedora_converter/default_work.rb

```diff
@@ -100,6 +100,9 @@ class DefaultWork < ActiveFedora::Base
       include Hydra::AccessControls::Embargoable
```

Comment:
Why not change this to `Hyrax::Embargoable` here as well instead of copying in the methods?